testdata/baselines/reference/submoduleAccepted/conformance/plainJSGrammarErrors.errors.txt.diff

```diff
@@ -1,69 +1,33 @@
 --- old.plainJSGrammarErrors.errors.txt
 +++ new.plainJSGrammarErrors.errors.txt
-@@= skipped -4, +4 lines =@@
- plainJSGrammarErrors.js(17,9): error TS18041: A 'return' statement cannot be used inside a class static block.
- plainJSGrammarErrors.js(20,5): error TS1089: 'static' modifier cannot appear on a constructor declaration.
- plainJSGrammarErrors.js(21,5): error TS1089: 'async' modifier cannot appear on a constructor declaration.
--plainJSGrammarErrors.js(22,5): error TS8009: The 'const' modifier can only be used in TypeScript files.
- plainJSGrammarErrors.js(22,11): error TS1248: A class member cannot have the 'const' keyword.
--plainJSGrammarErrors.js(23,5): error TS8009: The 'const' modifier can only be used in TypeScript files.
- plainJSGrammarErrors.js(23,11): error TS1248: A class member cannot have the 'const' keyword.
+@@= skipped -11, +11 lines =@@
  plainJSGrammarErrors.js(26,11): error TS1030: 'async' modifier already seen.
  plainJSGrammarErrors.js(28,11): error TS1029: 'static' modifier must precede 'async' modifier.
  plainJSGrammarErrors.js(29,5): error TS1031: 'export' modifier cannot appear on class elements of this kind.
 -plainJSGrammarErrors.js(29,5): error TS8009: The 'export' modifier can only be used in TypeScript files.
  plainJSGrammarErrors.js(30,5): error TS1031: 'export' modifier cannot appear on class elements of this kind.
++plainJSGrammarErrors.js(34,9): error TS8017: Signature declarations can only be used in TypeScript files.
```

Comment:
Seemingly a difference here.

Reply:
Probably inconsequential, given it's a missing body error in JS. Which, sort of funny that we can detect that there's a syntax error for JS, as that implies that this check could be elsewhere.

Reply:
We already have grammar checks for all modifiers, so the check for `export` in Strada is duplicative (as evident from the diagnostics). Also, it's not even correct, `export` _is_ permitted in JS.